src/Humanizer.Tests.Shared/Localisation/fr/TimeOnlyHumanizeTests.cs

```diff
@@ -0,0 +1,86 @@
+﻿#if NET6_0_OR_GREATER
+
+using System;
+using Humanizer.Configuration;
+using Humanizer.DateTimeHumanizeStrategy;
+using Xunit;
+
+namespace Humanizer.Tests.Localisation.fr
+{
+    [UseCulture("fr")]
+    public class TimeOnlyHumanizeTests
+    {
+        [Fact]
+        public void DefaultStrategy_SameTime()
+        {
+            Configurator.TimeOnlyHumanizeStrategy = new DefaultTimeOnlyHumanizeStrategy();
```

Comment:
Can we test this using overloads without needing to change the statics? I'm concerned this could make other tests unpredictable.

Reply:
It seems that changing statics was unnecessary after all, so I removed the problematic lines. However, this code was already present elsewhere:
- https://github.com/Humanizr/Humanizer/blob/937e827b8d664ce800b31160e51e8b2eca95fadc/src/Humanizer.Tests.Shared/TimeOnlyHumanizeTests.cs#L17
- https://github.com/Humanizr/Humanizer/blob/aa107c63fb1e4f0f79d22c6b126586fb3f38f420/src/Humanizer.Tests.Shared/Localisation/is/TimeOnlyHumanizeTests.cs#L22

Do you want me to make the same changes there too?

Reply:
NVM, those other occurrences are justified by the fact that in other tests they use `PrecisionTimeOnlyHumanizeStrategy` instead.